.github/workflows/trivy.yml

```diff
@@ -0,0 +1,43 @@
+name: dsp-appsec-trivy
+on: [pull_request]
+
+jobs:
+  appsec-trivy:
+    # Parse Dockerfile and build, scan image if a "blessed" base image is not used
+    name: DSP AppSec Trivy check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # fetch JDK
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11'
```

Comment:
FWIW, we're currently using 8 to build and run.  Don't know if that'll be an issue here